src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -708,6 +709,58 @@ public KojiImportResult importBuild( KojiImport importInfo, Iterable<Supplier<Im
         }, "Failed to retrieve list of tags for build: %s", buildId );
     }
 
+    /**
+     * Get tags giving a list of build Ids. This uses multicall and is much faster than calling listTags(id) one by one.
+     * @return A Map where the build Id is the key and tags as value ( a list ).
+     */
+    public Map<Integer, List<KojiTagInfo>> listTags( List<Integer> buildIds, KojiSessionInfo session )
+                    throws KojiClientException
+    {
+        MultiCallRequest multiCallRequest = new MultiCallRequest();
+        List<KojiMultiCallObj> multiCallObjs = new ArrayList<>();
+        multiCallRequest.setMultiCallObjs( multiCallObjs );
+
+        buildIds.forEach(( buildId ) -> {
+            KojiMultiCallObj callObj = new KojiMultiCallObj();
+            callObj.setMethodName( "listTags" );
```

Comment:
I'm thinking it might be good to have a constants class that had these method names, so they are kept consistent with the one-off @Request classes with the same method. It would take a lot to move all of those @Request method names over, but we could do the overlapping ones and just sort of make it a convention to migrate them into the constants class when we do something like this for the first time.

Reply:
Not a big deal. I will make a constants class for them. This work is much less onerous than what I have done to refactor other stuff.